src/Microsoft.ML.OnnxTransformer/OnnxCatalog.cs

```diff
@@ -40,14 +42,19 @@ public static class OnnxCatalog
         /// <summary>
         /// Create a <see cref="OnnxScoringEstimator"/>, which applies a pre-trained Onnx model to the input column.
         /// Input/output columns are determined based on the input/output columns of the provided ONNX model.
+        /// Please refer to <see cref="OnnxScoringEstimator"/> to learn more about the necessary dependencies,
+        /// and how to run it on a GPU.
         /// </summary>
         /// <remarks>
         /// The name/type of input columns must exactly match name/type of the ONNX model inputs.
         /// The name/type of the produced output columns will match name/type of the ONNX model outputs.
         /// </remarks>
         /// <param name="catalog">The transform's catalog.</param>
         /// <param name="modelFile">The path of the file containing the ONNX model.</param>
-        /// <param name="shapeDictionary">ONNX shape should be used to over those loaded from <paramref name="modelFile"/>.</param>
+        /// <param name="shapeDictionary">ONNX shapes to be used over those loaded from <paramref name="modelFile"/>.
```

Comment:
Would it be useful to add an example here? Or do we already have one in the samples?

Reply:
We don't have samples for this.
I guess I can make one, but for that I'll need to find a model that requires it and upload it somewhere outside ML.NET repo so that it can be added to the sample?

Reply:
In the meantime, do you have any other suggestion on the docs? maybe we can merge this PR just to see if this forces the staged docs to update... and add that sample on another PR

Reply:
In the meantime, do you have any other suggestion on the docs, @natke ? maybe we can merge this PR just to see if this forces the staged docs to update... and add that sample on another PR

Reply:
Ok, I'm pushing this now to see if this unblocks the staging docs.